src/CPUSolver.cpp

```diff
@@ -681,13 +681,13 @@ void CPUSolver::transportSweep() {
   if (_cmfd != NULL && _cmfd->isFluxUpdateOn())
     _cmfd->zeroCurrents();
 
-  /* Loop over azimuthal angle halfspaces */
-  for (int i=0; i < 2; i++) {
+  /* Loop over azimuthal angle and periodic track halfspaces */
+  for (int i=0; i < 6; i++) {
```

Comment:
Maybe it's just me, but I found that "2" made relatively good intuitive sense for the term "halfspaces" but do not think that "6" does. I know this PR has been merged in already and I didn't have time to review it beforehand. Perhaps the term "halfspaces" is a bit of a misnomer now? Plus, I always dislike seeing somewhat arbitrary values hard-coded in places like this.